cmd/ocsp-updater/main.go

```diff
@@ -402,6 +402,21 @@ func (updater *OCSPUpdater) revokedCertificatesTick(ctx context.Context, batchSi
 
 	var allPurgeURLs []string
 	for _, status := range statuses {
+		// It's possible that, if our ticks are fast enough (mainly in tests), we
+		// will get a status where the ocspLastUpdated == revokedDate and has already
```

Comment:
nit: I think  "we will get a status where the ocspLastUpdated == revokedDate and has already been revoked." is missing a word:
```suggestion
		// will get a certificate status where the ocspLastUpdated == revokedDate and the certificate has already
```